src/services/index.ts

```
@@ -0,0 +1,49 @@
+   */
+  static extractPlaylistId(input: string): string | null {
+    try {
+      const url = new URL(input);
+      const listParam = url.searchParams.get('list');
+      return listParam && this.isValidPlaylistId(listParam) ? listParam : null;
+    } catch {
+      // If it's not a URL, check if it's a valid playlist ID
+      return this.isValidPlaylistId(input.trim()) ? input.trim() : null;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)